public/app/features/explore/Wrapper.tsx

```diff
@@ -31,18 +31,16 @@ export class Wrapper extends Component<WrapperProps> {
 
     return (
       <div className="page-scrollbar-wrapper">
-        <CustomScrollbar autoHeightMin={'100%'}>
-          <div className="explore-wrapper">
+        <div className="explore-wrapper">
```

Comment:
This change highlighting is weird. I just removed `<CustomScrollbar/>`.

Reply:
Where is CustomScrollbar now then? Without it looks pretty bad in firefox on linux & ubuntu

Reply:
It is in `public/app/features/explore/Explore.tsx` (in each individual pane).

Reply:
@ivanahuckova at the top of the github-diff you can switch "ignore-whitespace: ON", then the diff looks better. i think the reason is that the indent changed for a lot of lines, causing the large visual diff.